CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### 1.18.2 Feb 27, 2025
```

Comment:
I feel like we maybe need to ditch version numbers. It's getting hazy what's actually getting to production and what's not. 

It may be another year before this app is shelved completely. I think in a future PR we should remove version numbers to lessen confusion (especially if we're going to implement pipelines at some point)

Reply:
Yes let's discuss this and make a plan.